Find a functionally equivalent code:android.media.Ringtone.stop:COMMENT
Method Modifier: public      
Comment:/**
 * Stops a playing ringtone.
 */

@@ -1,7 +1,7 @@
 {
     if (mLocalPlayer != null) {
         destroyLocalPlayer();
-    } else if (mAllowRemote) {
+    } else if (mAllowRemote && (mRemotePlayer != null)) {
         try {
             mRemotePlayer.stop(mRemoteToken);
         } catch (RemoteException e) {

